addons/account/models/res_partner_bank.py

```diff
@@ -32,6 +51,53 @@ def _check_journal_id(self):
             if len(bank.journal_id) > 1:
                 raise ValidationError(_('A bank account can belong to only one journal.'))
 
+    @api.constrains('allow_out_payment')
+    def _check_allow_out_payment(self):
+        """ Block enabling the setting, but it can be set to false without the group. (For example, at creation) """
+        for bank in self:
+            if bank.allow_out_payment:
+                if not self.user_has_groups('account.group_validate_bank_account'):
+                    raise ValidationError(_('You do not have the right to trust or un-trust a bank account.'))
+
+    @api.depends('partner_id', 'sanitized_acc_number', 'allow_out_payment')
+    def _compute_display_account_warning(self):
+        for bank in self:
+            if bank.allow_out_payment or not bank.sanitized_acc_number or bank.acc_type != 'iban':
+                bank.iban_warning = False
+                bank.money_transfer_warning = False
+                continue
+            bank_country = bank.sanitized_acc_number[:2]
+            if bank_country != bank.partner_id.country_id.code:
+                bank.iban_warning = f"""
+                    <span class="text-warning">Medium risk</span>: Iban <span class="fw-bold">{bank.acc_number}</span>
+                    is not from the same country than the partner ({bank.partner_id.country_id.name}).<br/>
+                    <a href="https://www.odoo.com/documentation/master/applications/finance/accounting/payables/pay/trusted_accounts.html">Check why it's risky.</a>
+                """
+            else:
+                bank.iban_warning = False
+
+            bank_institution_code = bank.sanitized_acc_number[4:7]
+            money_transfer_service_map = {
+                '967': 'Wise',
+                '977': 'Paynovate',
+                '974': 'PPS EU SA',
+            }
```

Comment:
Make it overwritable by `l10n_` modules
```suggestion
            money_transfer_service_map = self._get_bad_banks()
```